Emmanuelaaron/carBooking
# Rspec Test

- Added test user to verify
    - user creation
    - user log in
    - user is unique
    - token creation
    - token verification
- Add test for cars to verify
    - Cars creation
    - Return all cars
    - Cars delete
- Add test for reservation
   - Create a new reservation
   - get all cars and cities at once
